featurebyte/query_graph/node/metadata/operation.py

```diff
@@ -155,28 +157,23 @@ def clone_without_internal_nodes(
             node_kwargs["node_name"] = graph_node_name
             if hasattr(self, "transforms"):
                 node_kwargs["transforms"] = [graph_node_transform] if graph_node_transform else []
-        return self.clone(**node_kwargs, **kwargs)
+        return dataclasses.replace(self, **{**node_kwargs, **kwargs})
```

Comment:
@YS-L I updated clone function here as i found that this part changes the nested column into dictionary while working on updating `OperationStructure`.